src/components/SettingsEnableReceiverInvitationRetry.tsx

```diff
@@ -42,71 +45,88 @@ export const SettingsEnableSmsRetry = () => {
 
   const handleToggleChange = () => {
     // Default interval is 2 days
-    mutateAsync(organization.data.smsResendInterval === 0 ? 2 : 0);
+    mutateAsync(
+      organization.data.receiverInvitationResendInterval === 0 ? 2 : 0,
+    );
   };
 
-  const handleSmsRetrySubmit = (event: React.FormEvent<HTMLFormElement>) => {
+  const handleInvitationRetrySubmit = (
+    event: React.FormEvent<HTMLFormElement>,
+  ) => {
     event.preventDefault();
 
-    if (smsRetryInterval) {
-      mutateAsync(smsRetryInterval);
+    if (receiverInvitationRetryInterval) {
+      mutateAsync(receiverInvitationRetryInterval);
     }
   };
 
-  const handleSmsRetryReset = (event: React.FormEvent<HTMLFormElement>) => {
+  const handleInvitationRetryReset = (
+    event: React.FormEvent<HTMLFormElement>,
+  ) => {
     event.preventDefault();
     setIsEditMode(false);
-    setSmsRetryInterval(organization.data.smsResendInterval);
+    setReceiverInvitationRetryInterval(
+      organization.data.receiverInvitationResendInterval,
+    );
   };
 
   const renderContent = () => {
     return (
       <div className="SdpSettings">
         <div className="SdpSettings__row">
           <div className="SdpSettings__item">
-            <label className="SdpSettings__label" htmlFor="sms-retry">
-              Enable automatic SMS retry
+            <label
+              className="SdpSettings__label"
+              htmlFor="receiver-invitation-retry"
+            >
+              Enable automatic Message retry
```

Comment:
Do we need to capitalize Message here?

```suggestion
              Enable automatic message retry
```